pkg/updater/resultstore/resultstore.go

```diff
@@ -499,6 +625,16 @@ func testMethodLimit(tg *configpb.TestGroup) int {
 	return testMethodLimit
 }
 
+func testMethodRegex(tg *configpb.TestGroup) (matchMethods *regexp.Regexp, unmatchMethods *regexp.Regexp, matchMethodsErr error, unmatchMethodsErr error) {
```

Comment:
Think it's worth adding some simple test cases for this.

Reply:
Done